### DISCUSSION TITLE
小伙伴们有什么好的想法？

### DISCUSSION THREAD

**Discussion by liuzhihang** [liuzhihang/doc-view#6] [Ideas]
## 👋 Welcome!

关于 Doc View 你有什么想法，都可以在下面讨论！

**Comment by helloshisheng** [comment]
请求参数类型为枚举类会出现意料之外的信息。
会将一个字段转换成 code、value、ordinal、name等字段。

**Reply by liuzhihang** [reply]
收到，晚些时候我调试一下，修改修改
